modules/document_repository/ajax/FileUpload.php

```diff
@@ -0,0 +1,362 @@
+<?php
```

Comment:
Since this is a rewrite of the module, I think the ajax scripts removed and replaced with real LORIS endpoints that handle POST/GET/DELETE/etc HTTP requests, rather than grandfathering them in with the existing scripts until we have time to rewrite them.

For instance,  we should have URLs like `document_repository/files/$filename` with `GET/DELETE/PUT` handlers instead of this script.